src/DynamoPackages/Package.cs

```diff
@@ -31,6 +30,41 @@ public string Name
         }
     }
 
+    public class PackageLoadState
+    {
+        public enum StateTypes
```

Comment:
I think if all this is going to be public it would be nice if this class, and all enums had /// summarys.